src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -10,17 +10,15 @@ collect metrics and traces about incoming web requests. This instrumentation
 also collects traces from incoming gRPC requests using
 [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCore).
 
-**Note: This component is based on the OpenTelemetry semantic conventions for
-[metrics](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md)
+**Note: This component is based on the
+[v1.23](https://github.com/open-telemetry/semantic-conventions/tree/v1.23.0) of
+http semantic conventions for
+[metrics](https://github.com/open-telemetry/semantic-conventions/tree/v1.23.0/docs/http/http-metrics.md)
 and
-[traces](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md).
-These conventions are
-[Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/document-status.md),
-and hence, this package is a [pre-release](../../VERSIONING.md#pre-releases).
-Until a [stable
-version](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/telemetry-stability.md)
-is released, there can be breaking changes. You can track the progress from
-[milestones](https://github.com/open-telemetry/opentelemetry-dotnet/milestone/23).**
+[traces](https://github.com/open-telemetry/semantic-conventions/tree/v1.23.0/docs/http/http-spans.md).
```

Comment:
You may consider checking the *.cs files for additional links to the older conventions. We had them in several places when we got started on this.

Reply:
why do we need older conventions?

Reply:
Sorry, I misspoke. While working on this the last few months we referenced the specification that was available at the time. I meant to say, we should review the codebase to confirm that all the older links were replaced with the current 1.23

Reply:
I think we did that already. Will do the clean up if needed separately.